app/assets/stylesheets/components/results-list/_all.scss

```diff
@@ -0,0 +1 @@
+@import 'results_list';
```

Comment:
Tiny thing but filenames should match their component name so this should be `results-list.scss` rather than `results_list.scss'